docs/zh_cn/user_guides/useful_tools.md

```diff
@@ -1 +1,476 @@
-## 日志分析（待更新）
+## 日志分析
+
+`tools/analysis_tools/analyze_logs.py` 可利用指定的训练log文件绘制 loss/mAP 曲线图，
+当第一次运行前请先运行 `pip install seaborn` 安装必要依赖.
+
+```shell
+python tools/analysis_tools/analyze_logs.py plot_curve [--keys ${KEYS}] [--eval-interval ${EVALUATION_INTERVAL}] [--title ${TITLE}] [--legend ${LEGEND}] [--backend ${BACKEND}] [--style ${STYLE}] [--out ${OUT_FILE}]
+```
+
+![loss curve image](../../../resources/loss_curve.png)
```

Comment:
please check this image path whether is correct.

Reply:
Thanks for your modification, I have corrected all the mistakes. I think this image path is right because i can see it by Typora